query-connector/src/app/tests/integration/usergroup-management.test.ts

```diff
@@ -197,88 +244,88 @@ describe("User Group and Query Membership Tests", () => {
   /**
    * Tests adding queries to a user group.
    */
-  test("should add queries to a group", async () => {
-    const result = await addQueriesToGroup(TEST_GROUP_ID, [TEST_QUERY_ID]);
-    expect(result).toContain(TEST_QUERY_ID);
-    expect(result.length).toBe(1);
-
-    const updatedQueries: Query[] =
-      await getQueriesWithGroupStatus(TEST_GROUP_ID);
-    const members = updatedQueries.filter((query) =>
-      query.userGroupMemberships?.some((m) => m.is_member),
-    );
-
-    expect(members.length).toBe(1);
-    expect(members.some((query) => query.id === TEST_QUERY_ID)).toBe(true);
-    expect(members.some((query) => query.name === "Test Query")).toBe(true);
-  });
-
-  test("should not add duplicate queries to a group", async () => {
-    await addQueriesToGroup(TEST_GROUP_ID, [TEST_QUERY_ID]);
-    const result = await addQueriesToGroup(TEST_GROUP_ID, [TEST_QUERY_ID]);
-    expect(result).toEqual([]);
-  });
+  // test("should add queries to a group", async () => {
```

Comment:
did we mean to uncomment or delete these?

Reply:
added back after that whole testing fandango! 🎉